flask_pymongo/__init__.py

```diff
@@ -34,9 +34,15 @@
 from pymongo import uri_parser
 from werkzeug.wsgi import wrap_file
 import pymongo
+# DriverInfo was added in PyMongo 3.7
+try:
+    from pymongo.driver_info import DriverInfo
+except ImportError:
+    DriverInfo = None
 
 from flask_pymongo.helpers import BSONObjectIdConverter, JSONEncoder
 from flask_pymongo.wrappers import MongoClient
+from flask_pymongo._version import __version__
```

Comment:
Looks like there's a small import order issue causing the failure in the tests. 

[I100 Import statements are in the wrong order. 'from flask_pymongo._version import __version__' should be before 'from flask_pymongo.wrappers import MongoClient'](https://dev.azure.com/dcrosta/flask-pymongo/_build/results?buildId=643&view=logs&j=d7cfc920-a370-512b-e3a4-0de93961c53f&t=eb05bb45-0d5d-5561-5c16-080fe4d46d33&l=575)

Reply:
Thanks for the heads up. Pushed a fix to reorder these.